redis/utils.py

```diff
@@ -8,7 +8,13 @@
 
     # Only support Hiredis >= 3.0:
     HIREDIS_AVAILABLE = int(hiredis.__version__.split(".")[0]) >= 3
+    if not HIREDIS_AVAILABLE:
+        raise ImportError("hiredis package should be >= 3.0.0")
     HIREDIS_PACK_AVAILABLE = hasattr(hiredis, "pack_command")
```

Comment:
Can it actually happen in `hiredis-py` doesn't have the `pack` command with versions >=3?

Reply:
Check was removed